pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -922,8 +926,14 @@ message NetworkConfig {
     string pod_cidr = 1;
 }
 
+message ContainerResourcesConfig {
+    string container_id = 1;
```

Comment:
Do you think this field makes more sense as a member of the `RuntimeConfig` message instead?